harness/determined/util.py

```diff
@@ -129,6 +129,37 @@ def wrap_metrics(
         return {"metrics": metrics, "stop_requested": stop_requested, "invalid_hp": invalid_hp}
 
 
+def make_serializable(obj: Any) -> Any:
```

Comment:
does it just support numpy?

Reply:
It looks like it doesn't.

Reply:
ah ok. i'm not super familiar with the code and I was wondering why we need new code for this --  why can't we use `json_encode` below then?

Reply:
`json_encode` doesn't "just" support numpy

Reply:
I don't understand, it has the same exact code for supporting numpy as the code you added. It is what is already used to encode the metrics as sent through the harness websocket.

Reply:
Sorry, I gave a wrong explanation in my last comment. The requesting function `request.put()` requires the body to be a dictionary rather than a json string. I don't need the json encoding part `json_encode`.

Reply:
I'd rather have a `put_json` than write this code twice. Or at least pull out a common func?

Reply:
fixed